server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/model/MessageProperties.java

```diff
@@ -217,7 +217,8 @@ private static boolean outputProperty(MessageProperty p) {
     public enum ReadProfile {
         Metadata(0, FetchGroup.MINIMAL),
         Header(1, FetchGroup.HEADERS),
-        Full(2, FetchGroup.FULL_CONTENT);
+        Fast(2, FetchGroup.HEADERS),
```

Comment:
Given our new design, as the MessageViewFactory end up doing the read, I am not sure it makes sense any longer to be sprecifying the FetchGroup here. 

Technically that is wrong: Fast FetchGroup can be either Headers or Full, depending on "the projection content..."